src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -705,6 +725,13 @@ void ApplicationManagerImpl::OnHMIStartedCooperation() {
           hmi_apis::FunctionID::VehicleInfo_IsReady, *this));
   ManageHMICommand(is_ivi_ready);
 
+#ifdef SDL_REMOTE_CONTROL
+  utils::SharedPtr<smart_objects::SmartObject> is_rc_ready(
+      MessageHelper::CreateModuleInfoSO(hmi_apis::FunctionID::RC_IsReady,
+                                        *this));
+#endif
+  ManageHMICommand(is_rc_ready);
```

Comment:
if SDL_REMOTE_CONTROL is OFF, then compiling failed at this line

Reply:
@JackLivio issue should be fixed

Reply:
@JackLivio will check and fix all issues with RC=OFF